Update Packages

mattjdnv opened the pull request:
Update: Tomcat & Nodejs

Note the updates to the core only script are being done in #1833

sputnam commented:
Looks like a java install error on the build stack
```
### Installing Java 8...
==> default: Downloading jdk-8u144-linux-x64.tar.gz ....
==> default: Finished download of jdk-8u144-linux-x64.tar.gz
==> default: tar (child): ./jdk-8u144-linux-x64.tar.gz: Cannot open
==> default: : No such file or directory
==> default: tar (child): Error is not recoverable: exiting now
==> default: tar: Child returned status 2
==> default: tar: Error is not recoverable: exiting now
==> default: mv: cannot stat ‘./jdk1.8.0_144’
==> default: : No such file or directory
==> default: update-alternatives: error: alternative path /usr/lib/jvm/oracle_jdk8/jre/bin/java doesn't exist
==> default: update-alternatives: error: alternative path /usr/lib/jvm/oracle_jdk8/bin/javac doesn't exist
==> default: ### Done with Java 8 install...
```

mattjdnv commented:
@sputnam I think I fixed the JDK error. I edited the wget command and didn't see the option at the end of the line. I removed the option so it should be good now.

jbronn commented:
The [nodesource repository](https://rpm.nodesource.com/pub/el/7/x86_64/) provides no version of 0.10.48 on CentOS 7 -- the maximum version available is 0.10.46.
